ui.content/src/main/content/META-INF/vault/filter.xml

```diff
@@ -27,24 +28,18 @@
         <include pattern="/etc/acs-commons/lists/report-parameter-components/jcr:content/list/default_\d+"/>
         <include pattern="/etc/acs-commons/notifications"/>
         <include pattern="/etc/acs-commons/notifications/jcr:content"/>
+        <include pattern="/etc/acs-commons/notifications/rep:policy(/.*)?"/>
         <include pattern="/etc/acs-commons/packagers"/>
         <include pattern="/etc/acs-commons/packagers/jcr:content"/>
         <include pattern="/etc/acs-commons/redirect-maps"/>
         <include pattern="/etc/acs-commons/redirect-maps/jcr:content"/>
+        <include pattern="/etc/acs-commons/redirect-maps/rep:policy(/.*)?" />
         <include pattern="/etc/acs-commons/exporters"/>
         <include pattern="/etc/acs-commons/exporters/jcr:content"/>
         <include pattern="/etc/acs-commons/qr-code"/>
-        <include pattern="/etc/acs-commons/qr-code/jcr:content"/>
-        <include
-            pattern="/etc/acs-commons/qr-code/jcr:content/clientlib-author"
-            mode="merge"/>
-        <exclude
```

Comment:
Any idea if this was here for a good reason?  i.e. does the QR Code tool store configs (set from AEM) under here and are we blowing them away on re-deploy of ACS Commons?

Reply:
I think indeed the QR code can be toggled (i.e. enabled/disabled). This settings is persisted in  /etc/acs-commons/qr-code/jcr:content/. So I think this part should be added with a dedicated merge. I will work on this.

Reply:
Should be fixed now

Reply:
@HitmanInWis this theoretical situation can easily be made into an automated test by an oakpal scan with a forcedRoot of `/etc/acs-commons/qr-code/jcr:content/config/oldNode` and the `basic/paths` check.